src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,12 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
+            {
+                Reporter.Error.WriteLine($"Creating new {languageName} project failed, project.json already exists.");
```

Comment:
@blackdwarf Can you review the recent changes.. The message indicates why the project creation failed with exactly which filename that's conflicting. Since the concept of a project is very hazy, is a collection of code files under a directory project, or does the presence of project.json make it a project..